docs/source/developer/quorum-ansible.md

```diff
@@ -0,0 +1,243 @@
+* Generate node tm keys
+* Copy the crypto material to Vault
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/crypto/tessera/Readme.md) for detailed information.
+
+## **create/crypto/tessera**
+
+This role creates genesis nodekey.
+* Check if nodekey is present in vault
+* Call nested check for each node
+* Check if nodekey already present in the vault
+* vault_check variable
+* Fetching data of validator nodes in the network from network.yaml
+* Get validator node data
+* Create build directory if it does not exist
+* Generate istanbul files
+* Rename the directories created above with the elements of validator_node_list
+ * Delete the numbered directories
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/genesis_nodekey/Readme.md) for detailed information.
+
+## **create/k8_component**
+
+This role creates deployment files for nodes, namespace storageclass, service accounts and clusterrolebinding. Deployment file for a node is created in a directory with name=nodeName, nodeName is stored in component_name
+* "Ensures {{ release_dir }}/{{ component_name }} dir exists"
+* create {{ component_type }} file for {{ component_name }}
+* Helm lint
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/k8_component/Readme.md) for detailed information.
+
+## **create/namespace_serviceaccount**
+
+This role creates the deployment files for namespaces, vault-auth, vault-reviewer and clusterrolebinding for each node
+* Check if namespace exists
+* Create namespace for {{ organisation }}
+* Create vault auth service account for {{ organisation }}
+* Create vault reviewer for {{ organisation }}
+* Create clusterrolebinding for {{ organisation }}
+* Push the created deployment files to repository
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/namespace_serviceaccount/Readme.md) for detailed information.
+
+
+## **create/storageclass**
+
+This role creates value files for storage class
+* Check if storageclass exists
+* Create storageclass
+* Push the created deployment files to repository
+* Wait for Storageclass creation for {{ component_name }}
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/storageclass/Readme.md) for detailed information.
+
+## **create/tessera**
+
+* Set enode_data_list to []
+* Get enode data for each node of all organization
+* Get enode data
+* Get enode_data
+* Get validator node data
+
+Follow [Readme](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/quorum/configuration/roles/create/tessera/Readme.md) for detailed information.
+
```

Comment:
Add the "delete" roles as well, this is available on develop now. Also include reset_network

Reply:
Added